cmd/agent/agent.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/loki"
+	"github.com/grafana/agent/pkg/tempo"
+
+	"github.com/grafana/agent/pkg/config"
+	"github.com/grafana/agent/pkg/prom"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/weaveworks/common/server"
+
+	"github.com/go-kit/kit/log"
+)
+
+type EntryPoint struct {
```

Comment:
Smallest nit: I'd treat entrypoint as one word
```suggestion
type Entrypoint struct {
```